catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -0,0 +1,259 @@
+import { respondBetween, respondTo } from '@weco/common/views/themes/mixins';
+import CataloguePageLayout from 'components/CataloguePageLayout/CataloguePageLayout';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { FunctionComponent, ReactElement, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { pageDescriptions } from '@weco/common/data/microcopy';
+
+const NavBar = styled.nav`
+  border-bottom: 1px solid ${props => props.theme.color('warmNeutral.400')};
+  [aria-current='page'] {
+    &:after {
+      width: 100%;
+    }
+  }
+`;
+const NavList = styled.ul`
+  display: flex;
+  list-style: none;
+  list-style-position: inside;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+`;
+const NavItem = styled.li`
+  ${respondTo(
+    'headerMedium',
+    `
+    border-bottom: 0;
+    margin-right: calc(3vw - 20px);
+  `
+  )}
+
+  ${respondBetween(
+    'headerMedium',
+    'large',
+    `
+    font-size: 1.5vw;
+  `
+  )}
+
+  ${respondTo(
+    'xlarge',
+    `
+    margin-right: 1.4rem;
+  `
+  )}
+`;
+const NavLink = styled.a`
+  padding: 1.4rem 0.3rem;
+  display: inline-block;
+  text-decoration: none;
+  position: relative;
+  z-index: 1;
+  transition: color 400ms ease;
+  white-space: nowrap;
+
+  ${respondTo(
+    'headerMedium',
+    `
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  `
+  )}
+
+  &:after {
+    content: '';
+    position: absolute;
+    bottom: 0.1rem;
+    height: 0.2rem;
+    left: 0;
+    width: 0;
+    background: ${props => props.theme.color('yellow')};
+    z-index: -1;
+    transition: width 200ms ease;
+
+    ${respondTo(
```

Comment:
Ohh maybe I've missed this when I refactored media-queries, but we're aiming to use [this method](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/common/views/components/Footer/FooterNav.tsx#L24) moving forward. We used to have a few different ways to declare and I think this one might've slipped between cracks.